libs/entry-components/search-filter/public-api.ts

```diff
@@ -1,8 +1,11 @@
 export { EntrySearchFilterComponent } from './entry-search-filter.component';
 
-export { SearchFilterParams } from './search-filter-params';
-export { SearchFilterBase } from './search-filter-base';
-export { SearchFilterInput } from './search-filter-textbox';
+export { SearchFilterParams } from './search-filter-params.type';
+export { SearchFilterBase } from './search-filter-input/search-filter-base.model';
+export { TextSearchFilter } from './search-filter-input/inputs/text-search-filter.model';
+export { SelectSearchFilter } from './search-filter-input/inputs/select-search-filter.model';
+export { DynamicSelectSearchFilter } from './search-filter-input/inputs/dynamic-select-search-filter.model';
+export { SelectSearchFilterOption } from './search-filter-input/inputs/select-search-filter-option.model';
```

Comment:
Regarding this model. We have SelectOption inside entry-form component that can be re-used here as well, but it is in different lib.

Would it make sense to re-use autocomplete component here?